insights/parsers/smartctl.py

```diff
@@ -1,7 +1,13 @@
 """
-SMARTctl - command ``/sbin/smartctl -a {device}``
-=================================================
+smartctl - parser for smartctl commands
+=======================================
+
+Classes to parse the output of smartctl commands:
+
```

Comment:
Shall we also include the another parser `SMARTctl` here?

Reply:
@wushiqinlou - would you mind add the SMARTctl here?